pages/linux/zypper.md

```diff
@@ -1,6 +1,7 @@
 # zypper
 
 > SUSE & openSUSE package management utility.
+> More information: <https://en.opensuse.org/SDB:Zypper_manual>.
 
 - Synchronize list of packages and versions available:
```

Comment:
@victorhck do you want to update this example description?

Reply:
> @victorhck do you want to update this example description?

Wich one?

Reply:
Line 6

Reply:
I'm a openSUSE user, but not a English speaker, so please fix if needed! :)
Maybe the current text can be replaced with:

> Download and refresh the metadata of repository list

Reply:
If I check the pages `apt` and `pacman-sync` the current example description is fine for us, but we can hear other opinions.

Reply:
I'm a bit confused. What about the existing description needs changing?

Reply:
> If I check the pages `apt` and `pacman-sync` the current example description is fine for us, but we can hear other opinions.

> I'm a bit confused. What about the existing description needs changing?

So, let's left the text as is. Ok for me too...